.github/workflows/benchmarks.yml

```diff
@@ -51,8 +51,29 @@ jobs:
           alert-comment-cc-users: '@runwasi-committers'
           # Enable Job Summary
           summary-always: true
-          # Where the previous data file is stored
-          external-data-json-path: ./cache/benchmark-data.json
+          # Automatically push the benchmark result to gh-pages branch
+          # See https://github.com/benchmark-action/github-action-benchmark?tab=readme-ov-file#charts-on-github-pages-1 for more details
+          auto-push: true
```

Comment:
Do we want to push these to results on PR's  or maybe it should be to a different branch?

Reply:
That would be nice to have but the action itself doesn't support it. We can use `gh` CLI to do this, perhaps.

Reply:
Ok, Maybe for now we should only enable pushing results on merge to the main branch?

Reply:
Updated